ast-visual-studio-extension/CxExtension/Services/ASCAService.cs

```diff
@@ -60,30 +59,28 @@ private async void OnDebounceTimerElapsed(object sender, ElapsedEventArgs e)
                     var content = textDocument.StartPoint.CreateEditPoint().GetText(textDocument.EndPoint);
                     if (textDocument?.StartPoint == null || textDocument?.EndPoint == null)
                     {
-                        return; 
+                        return;
                     }
                     var originalFileName = Path.GetFileName(document.FullName);
                     var timestamp = DateTime.Now.ToString("yyyyMMdd_HHmmss");
-                    tempFilePath = Path.Combine(Path.GetTempPath(), $"{originalFileName}_{timestamp}.cs");
+                    var fileNameWithoutExt = Path.GetFileNameWithoutExtension(originalFileName);
+                    var extension = Path.GetExtension(originalFileName);
```

Comment:
does extension var includes "." before the extension itself?

Reply:
Yes